pkg/utils/api/image.go

```diff
@@ -90,6 +92,28 @@ func extract(obj interface{}, path []string, keyPath, valuePath string, fields [
 		if !ok {
 			return fmt.Errorf("invalid value")
 		}
+
+		if jmesPath != "" {
+			jmesPath = strings.ReplaceAll(jmesPath, "$image", fmt.Sprintf(`'%s'`, value))
+
+			jp, err := jmespath.New(jmesPath)
+			if err != nil {
+				return fmt.Errorf("invalid jmespath %s: %v", jmesPath, err)
+			}
+
+			result, err := jp.Search("")
```

Comment:
why not `result, err := jp.Search(value)` ?

Reply:
the jmespath filter would be `trim_prefix(@, 'docker://')`

Reply:
Thanks, this is much cleaner. Updated to this.